packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -461,3 +464,23 @@ export class DefaultCrudRepository<
     return models.map(m => this.toEntity<R>(m));
   }
 }
+
+/**
+ * Default implementation of CRUD repository using legacy juggler model
+ * and data source with beginTransaction() method for connectors which
+ * support Transactions
+ */
+
+export class DefaultTransactionalRepository<
```

Comment:
Is it necessary to introduce a new repository class? Can we change `DefaultCrudRepository` class to implement `TransactionalRepository` too?

I can see benefits of having `DefaultCrudRepository` and `DefaultTransactionalRepository`, but it also have downsides.

@raymondfeng WDYT?

Reply:
I am also okay with either approach, but this felt cleaner as I was writing the docs. Later on, we can decide to add it to our CLI templates too.